tests/rkt_run_pod_manifest_test.go

```diff
@@ -603,5 +603,10 @@ func TestPodManifest(t *testing.T) {
 			}
 		}
 		verifyHostFile(t, tmpdir, "file", i, tt.expectedResult)
+
+		// Deleting testing images here to free some space, otherwise Semaphore CI might error.
+		for _, v := range tt.images {
+			os.Remove(v.name)
```

Comment:
Yea, but I didn't save the name at the time.
Thanks for reminding though, actually I found I should add parent path to the names...